test/test-functions

```diff
@@ -632,6 +633,13 @@ install_keymaps() {
     done
 }
 
+install_zoneinfo() {
+    for i in /usr/share/zoneinfo/{,*/,*/*/}*; do
```

Comment:
In theory, it can take up a lot of space, which might be a problem given that the size of the image we create is about 500Mb. Do we need the whole database to simply run the unit tests?